tests/integration/goldens/asset/google/cloud/asset_v1/services/asset_service/transports/rest.py

```diff
@@ -632,7 +632,7 @@ class AssetServiceRestStub:
     _interceptor: AssetServiceRestInterceptor
 
 
-class AssetServiceRestTransport(AssetServiceTransport):
+class AssetServiceRestTransport(_BaseAssetServiceRestTransport):
     """REST backend transport for AssetService.
```

Comment:
```suggestion
    """REST backend synchronous transport for AssetService.
```

(and we'll say "asynchronous" for the async sibling classes)

Also, I think we can remove the "beta" paragraph at the end of this comment.

Reply:
addressed.

Reply:
Don't see the change in the diff.